tests/core/datatypes/CIEC_LTIME_test.cpp

```diff
@@ -588,10 +588,10 @@ BOOST_AUTO_TEST_CASE(parse_missing_s_in_microseconds_literal)
 {
   CIEC_LTIME time;
 
-  BOOST_CHECK_EQUAL(time.fromString("LT#-1u"), -1);
+  BOOST_CHECK_EQUAL(time.fromString("LT#-1u"), -1_LTIME);
   BOOST_CHECK_EQUAL(time.getInMicroSeconds(), 0);
 
-  BOOST_CHECK_EQUAL(time.fromString("LT#-1ut"), -1);
+  BOOST_CHECK_EQUAL(time.fromString("LT#-1ut"), -1_LTIME);
```

Comment:
@azoitl shouldn't those tests fail? I mean neither -1u nor -1ut are valid time literals

Reply:
@MartinMelikMerkumians fromstring is returing an int with -1 one on failure or the number of used bytes on success. 

@mhansal why have you changed that to a time literal?

Reply:
Oh, I see. Yeah, the the comparison values are wrong. I thought I disabled the implicit conversion to builtin types from LTIME

Reply:
Fixed.